com.coeck_studios.Dinonuggys-Journey.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
Ideally, this should be maintained upstream.

Reply:
[Done](https://codeberg.org/CoEck/Dinonuggys-Journey/src/branch/main/com.coeck_studios.Dinonuggys-Journey.desktop)